charts/openebs/values.yaml

```diff
@@ -220,30 +230,16 @@ webhook:
     #   cpu: 250m
     #   memory: 500Mi
 
-jiva:
-  image: "openebs/jiva"
-  imageTag: "2.9.0"
-  replicas: 3
-  defaultStoragePath: "/var/openebs"
```

Comment:
@shubham14bajpai in jiva csi how we are passing this value?
Is this a local pv config in jiva?

Reply:
This is not required in jiva csi as there is a policy which takes openebs-hostpath as replica storageclass